### DISCUSSION TITLE
0.12.1 - Fix Settings Import

### DISCUSSION THREAD

**Discussion by Timthreetwelve** [Timthreetwelve/GetMyIP#120] [Announcements]
### This update includes a new Language translation, and a fix to a settings import issue.
>[!Note]
> I'm sorry for another release so soon after releasing 0.12.1, but I wanted to get the fix for #114 released.

#### Changes: ✨

- New: 🎉 Japanese (ja-JP) language translation. Thanks to @coolvitto. #115 🎉
- Fixed: Import of settings, introduced in version 0.12.1, was not reliably restarting the application after import. #114
- Fixed: Improper display of translation percentage on About page.
- Updated: French language updates. Thanks, @logounet. #105
- Updated: Korean language updates. Thanks, @VenusGirl. #118
- Updated: Italian language updates. Thanks, @bovirus. #116
- Updated: Dutch language updates. Thanks, @CMTriX.
- Updated: Resiliency improvements and code optimization.

### Translations ✍🏻

My thanks to all the people that have contributed translations and updates.
> [!Important]
> If you have previously contributed, please take a moment to verify that your translation is up to date. Thanks!

If you wish to contribute a translation, or update a current translation, check out the [Contribute a Translation](https://github.com/Timthreetwelve/GetMyIP/wiki/Contribute-a-Translation) topic in the Wiki. Your help would be awesome.

**Full Changelog**: <https://github.com/Timthreetwelve/GetMyIP/compare/v0.12.1...v0.12.2>

#### File Descriptions

| File Name                            | Description                                                                  | Size  |
|--------------------------------------|------------------------------------------------------------------------------|------:|
| GetMyIP_0.12.2.0_x64x86_Setup.exe    | Installer, **requires** .NET 8 runtime to already be present                 |9.17 MB|
| GetMyIP_0.12.2.0_SC_x64_Setup.exe    | Installer, x64 architecture, Self-Contained, .NET runtime **not required**   |61.94 MB|
| GetMyIP_0.12.2.0_SC_x86_Setup.exe    | Installer, x86 architecture, Self-Contained, .NET runtime **not required**   |57.22 MB|
| GetMyIP_0.12.2.0_x64x86_Portable.zip | Zip archive, **requires** .NET 8 runtime to already be present               |13.13 MB|
| GetMyIP_0.12.2.0_SC_x64_Portable.zip | Zip archive, x64 architecture, Self-Contained, .NET runtime **not required** |85.54 MB|
| GetMyIP_0.12.2.0_SC_x86_Portable.zip | Zip archive, x86 architecture, Self-Contained, .NET runtime **not required** |79.60 MB|

#### Checksums

| File Name                            | SHA256 Hash | Virus Total |
|--------------------------------------|-------------|:-----------:|
| GetMyIP_0.12.2.0_x64x86_Setup.exe    | `6127d7ef22b454d2baf1e3e066d6c3bf3308f411f4b36593c3e3baac7f3bea15`| [Link](https://www.virustotal.com/gui/file/6127d7ef22b454d2baf1e3e066d6c3bf3308f411f4b36593c3e3baac7f3bea15) |
| GetMyIP_0.12.2.0_SC_x64_Setup.exe    | `5e29a31856829eb77a9eaa81b08c4af4f347a3ec36d70d67b72439dc9d0bde2e`| [Link](https://www.virustotal.com/gui/file/5e29a31856829eb77a9eaa81b08c4af4f347a3ec36d70d67b72439dc9d0bde2e)
| GetMyIP_0.12.2.0_SC_x86_Setup.exe    | `d05560f37323dd27bcd5d1e1d6169d58e7ebc73ab3cd552f327e555cbc2abab1`| [Link](https://www.virustotal.com/gui/file/d05560f37323dd27bcd5d1e1d6169d58e7ebc73ab3cd552f327e555cbc2abab1) |
| GetMyIP_0.12.2.0_x64x86_Portable.zip | `9c2785ad3fe284da2921dfb5fc36d16a6053886a6fa87925137a04682ff484a6`| [Link](https://www.virustotal.com/gui/file/9c2785ad3fe284da2921dfb5fc36d16a6053886a6fa87925137a04682ff484a6) |
| GetMyIP_0.12.2.0_SC_x64_Portable.zip | `a9fa3fbf39ff66bba30f501c8d3f9e19ec6f39427bafd130d6818f6193685bc0`| [Link](https://www.virustotal.com/gui/file/a9fa3fbf39ff66bba30f501c8d3f9e19ec6f39427bafd130d6818f6193685bc0) |
| GetMyIP_0.12.2.0_SC_x86_Portable.zip | `9bd293019270f870e356332064a7ae81bdba91916e6eeda153ebf3fe777194f7`| [Link](https://www.virustotal.com/gui/file/9bd293019270f870e356332064a7ae81bdba91916e6eeda153ebf3fe777194f7)|

<hr /><em>This discussion was created from the release <a href='https://github.com/Timthreetwelve/GetMyIP/releases/tag/v0.12.2'>0.12.1 - Fix Settings Import</a>.</em>

**Comment by coolvitto** [comment]
Hello
Thank you for the quick update.

I checked the Japanese display and it seems fine.

coolvitto
________________________________
差出人: Tim Kennedy ***@***.***>
送信日時: 2024年11月28日 8:15
宛先: Timthreetwelve/GetMyIP ***@***.***>
CC: coolvitto ***@***.***>; Mention ***@***.***>
件名: [Timthreetwelve/GetMyIP] 0.12.1 - Fix Settings Import (Discussion #120)


This update includes a new Language translation, and a fix to a settings import issue.

Note

I'm sorry for another release so soon after releasing 0.12.1, but I wanted to get the fix for #114<https://github.com/Timthreetwelve/GetMyIP/issues/114> released.

Changes: ✨

  *   New: 🎉 Japanese (ja-JP) language translation. Thanks to @coolvitto<https://github.com/coolvitto>. #115<https://github.com/Timthreetwelve/GetMyIP/discussions/115> 🎉
  *   Fixed: Import of settings, introduced in version 0.12.1, was not reliably restarting the application after import. #114<https://github.com/Timthreetwelve/GetMyIP/issues/114>
  *   Fixed: Improper display of translation percentage on About page.
  *   Updated: French language updates. Thanks, @logounet<https://github.com/logounet>. #105<https://github.com/Timthreetwelve/GetMyIP/discussions/105>
  *   Updated: Korean language updates. Thanks, @VenusGirl<https://github.com/VenusGirl>. #118<https://github.com/Timthreetwelve/GetMyIP/pull/118>
  *   Updated: Italian language updates. Thanks, @bovirus<https://github.com/bovirus>. #116<https://github.com/Timthreetwelve/GetMyIP/pull/116>
  *   Updated: Dutch language updates. Thanks, @CMTriX<https://github.com/CMTriX>.
  *   Updated: Resiliency improvements and code optimization.

Translations ✍🏻

My thanks to all the people that have contributed translations and updates.

Important

If you have previously contributed, please take a moment to verify that your translation is up to date. Thanks!

If you wish to contribute a translation, or update a current translation, check out the Contribute a Translation<https://github.com/Timthreetwelve/GetMyIP/wiki/Contribute-a-Translation> topic in the Wiki. Your help would be awesome.

Full Changelog: v0.12.1...v0.12.2<https://github.com/Timthreetwelve/GetMyIP/compare/v0.12.1...v0.12.2>

File Descriptions
File Name       Description     Size
GetMyIP_0.12.2.0_x64x86_Setup.exe       Installer, requires .NET 8 runtime to already be present        9.17 MB
GetMyIP_0.12.2.0_SC_x64_Setup.exe       Installer, x64 architecture, Self-Contained, .NET runtime not required  61.94 MB
GetMyIP_0.12.2.0_SC_x86_Setup.exe       Installer, x86 architecture, Self-Contained, .NET runtime not required  57.22 MB
GetMyIP_0.12.2.0_x64x86_Portable.zip    Zip archive, requires .NET 8 runtime to already be present      13.13 MB
GetMyIP_0.12.2.0_SC_x64_Portable.zip    Zip archive, x64 architecture, Self-Contained, .NET runtime not required        85.54 MB
GetMyIP_0.12.2.0_SC_x86_Portable.zip    Zip archive, x86 architecture, Self-Contained, .NET runtime not required        79.60 MB
Checksums
File Name       SHA256 Hash     Virus Total
GetMyIP_0.12.2.0_x64x86_Setup.exe       6127d7ef22b454d2baf1e3e066d6c3bf3308f411f4b36593c3e3baac7f3bea15        Link<https://www.virustotal.com/gui/file/6127d7ef22b454d2baf1e3e066d6c3bf3308f411f4b36593c3e3baac7f3bea15>
GetMyIP_0.12.2.0_SC_x64_Setup.exe       5e29a31856829eb77a9eaa81b08c4af4f347a3ec36d70d67b72439dc9d0bde2e        Link<https://www.virustotal.com/gui/file/5e29a31856829eb77a9eaa81b08c4af4f347a3ec36d70d67b72439dc9d0bde2e>
GetMyIP_0.12.2.0_SC_x86_Setup.exe       d05560f37323dd27bcd5d1e1d6169d58e7ebc73ab3cd552f327e555cbc2abab1        Link<https://www.virustotal.com/gui/file/d05560f37323dd27bcd5d1e1d6169d58e7ebc73ab3cd552f327e555cbc2abab1>
GetMyIP_0.12.2.0_x64x86_Portable.zip    9c2785ad3fe284da2921dfb5fc36d16a6053886a6fa87925137a04682ff484a6        Link<https://www.virustotal.com/gui/file/9c2785ad3fe284da2921dfb5fc36d16a6053886a6fa87925137a04682ff484a6>
GetMyIP_0.12.2.0_SC_x64_Portable.zip    a9fa3fbf39ff66bba30f501c8d3f9e19ec6f39427bafd130d6818f6193685bc0        Link<https://www.virustotal.com/gui/file/a9fa3fbf39ff66bba30f501c8d3f9e19ec6f39427bafd130d6818f6193685bc0>
GetMyIP_0.12.2.0_SC_x86_Portable.zip    9bd293019270f870e356332064a7ae81bdba91916e6eeda153ebf3fe777194f7        Link<https://www.virustotal.com/gui/file/9bd293019270f870e356332064a7ae81bdba91916e6eeda153ebf3fe777194f7>
________________________________
This discussion was created from the release 0.12.1 - Fix Settings Import<https://github.com/Timthreetwelve/GetMyIP/releases/tag/v0.12.2>.

**Reply by Timthreetwelve** [reply]
Thank you for checking.

Tim
